src/Npgsql/Util/VersionExtensions.cs

```diff
@@ -12,6 +12,6 @@ public static bool IsGreaterOrEqual(this Version version, int major, int minor,
                 ? version.Major > major
                 : version.Minor != minor
                     ? version.Minor > minor
-                    : version.Build >= build;
+                    : version.Build == -1 || version.Build >= build;
```

Comment:
Should we even look at the build? We always seem to pass 0 in the current codebase, and IIRC since version 10 PG no longer has three version elements.

Reply:
Pretty sure we can drop it, just didn't want to change it more than necessary.